examples/mongodb/2_load_and_index.ts

```diff
@@ -34,6 +34,7 @@ async function loadAndIndex() {
     dbName: databaseName,
     collectionName: vectorCollectionName, // this is where your embeddings will be stored
     indexName: indexName, // this is the name of the index you will need to create
+    populatedMetadataFields: ["_node_type", "document_id"], // this is the field that will be used for the query
```

Comment:
how about calling it `indexedMetadataFields`?
and to make the example easier to understand, we'll add some metadata to each document like `content_type='tweet`?
```suggestion
    indexedMetadataFields: ["content_type"], // this is the field that will be used for the query
```